src/client/pages/ConsentsData.tsx

```diff
@@ -1,34 +1,85 @@
 import React from 'react';
 import { css } from '@emotion/react';
-import { neutral, space, textSans } from '@guardian/source-foundations';
-import { getAutoRow, gridItemColumnConsents } from '@/client/styles/Grid';
+import {
+  neutral,
+  space,
+  remSpace,
+  textSans,
+} from '@guardian/source-foundations';
+import {
+  getAutoRow,
+  gridItemColumnConsents,
+  gridItemToggleSwitch,
+} from '@/client/styles/Grid';
 import { CONSENTS_PAGES } from '@/client/models/ConsentsPages';
 import {
   heading,
   text,
-  headingMarginSpace6,
+  textBold,
   greyBorderTop,
 } from '@/client/styles/Consents';
-import { Checkbox, CheckboxGroup } from '@guardian/source-react-components';
 import { ConsentsLayout } from '@/client/layouts/ConsentsLayout';
-import { Consents } from '@/shared/model/Consent';
-
 import { ConsentsContent } from '@/client/layouts/shared/Consents';
+import { ExternalLink } from '../components/ExternalLink';
+import locations from '@/shared/lib/locations';
+import { ToggleSwitchInput } from '../components/ToggleSwitchInput';
 
 type ConsentsDataProps = {
   consented?: boolean;
   description?: string;
+  name?: string;
+  id: string;
 };
 
-const fieldset = css`
+const switchRow = css`
   border: 0;
   padding: 0;
   margin: ${space[4]}px 0 0 0;
   ${textSans.medium()}
 `;
 
-const checkboxLabel = css`
-  color: ${neutral[7]};
+const removeMargin = css`
+  margin: 0;
+`;
+
+const toggleSwitchAlignment = css`
+  justify-content: space-between;
+  span {
+    align-self: flex-start;
+    margin-top: 4px;
+  }
+`;
+
+const listBullets = css`
+  list-style: none;
+  line-height: ${remSpace[6]};
+  padding-left: 0;
+  text-indent: -18px; // second line indentation
+  margin-left: 18px; // second line indentation
+  li {
+    font-size: 17px;
+  }
+  li:first-of-type {
+    margin-top: ${space[2]}px;
+  }
+  // ::marker is not supported in IE11
```

Comment:
css comments user `/* .. */` syntax, using `//` will be parsed as invalid syntax and may cause other issues

```suggestion
  /* ::marker is not supported in IE11 */
```

Reply:
did not know that. thanks!

Reply:
https://css-tricks.com/snippets/css/comments-in-css/